src/cascadia/TerminalConnection/Resources/en-US/Resources.resw

```diff
@@ -207,6 +207,10 @@
     <value>[process exited with code {0}]</value>
     <comment>The first argument {0} is the error code of the process. When there is no error, the number ZERO will be displayed. </comment>
   </data>
+  <data name="CtrlDToClose" xml:space="preserve">
+    <value>You can now close this terminal with Ctrl+D or Enter to restart.</value>
+	<comment>{Locked="Ctrl+D","Enter"} These represent keys the user will press (control+D and Enter).</comment>

```

Comment:
nit: We may want to let them translate Ctrl as in German it may be abbreviated `Strg`. However, we already aren't good about localizing `Ctrl`